backend/app/tests/utils/json/test_json_loader.py

```diff
@@ -0,0 +1,27 @@
+import os 
+from app.utils.json import json_loader
+
+def test_successfully_load_data() -> None:
+    """
+    Test to check that the json_loader correctly reads in the Wikipedia output data.
+    """
+
+    file_path = os.getcwd()+"\\app\\tests\\data\\test-mini-wikipedia.output.txt"
+    wikipedia_data = json_loader.load_data(file_path)[0]
+
+    assert list(wikipedia_data.keys())[0] == "Amphibian"
+
+
+def test_unsuccessfully_load_data_with_incorrect_file_path() -> None:
```

Comment:
I don't know what these tests are supposed to be, but you could probably use pytest's parameterized test support rather than having multiple test methods.

Reply:
I am using paramaterized fixtures to test ` json_loader`.